main.py

```diff
@@ -64,6 +64,7 @@ def add_commands(up: Updater) -> None:
         BotCommand("cloud", "risorse didattiche in cloud"),
         BotCommand("regolamentodidattico", "lista dei regolamenti didattici"),
         BotCommand("ricevimenti", "lista orari ricevimenti dei professori"),
+        BotCommand("syllabus", "lista e informazioni pe rogni materia di ogni corso di laurea"), 
```

Comment:
let's remove trailing whitespace
```suggestion
        BotCommand("syllabus", "lista e informazioni pe rogni materia di ogni corso di laurea"),
```